src/main/java/pl/noip/lolstats/lol/stats/jwt/JwtGenerator.java

```diff
@@ -3,5 +3,5 @@
 public interface JwtGenerator {
```

Comment:
Maybe change JwtGenerator interface to
String generate(Account account);
instead of many methods with different arguments.